plugins/source/launchdarkly/resources/services/auditlog/auditlog_entries.go

```diff
@@ -0,0 +1,87 @@
+package auditlog
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/cloudquery/cloudquery/plugins/source/launchdarkly/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+	ldapi "github.com/launchdarkly/api-client-go/v11"
+)
+
+func AuditLogEntries() *schema.Table {
+	return &schema.Table{
+		Name:        "launchdarkly_auditlog_entries",
+		Description: `https://apidocs.launchdarkly.com/tag/Audit-log#operation/getAuditLogEntries`,
+		Resolver:    fetchAuditLogEntries,
+		Transform:   transformers.TransformWithStruct(&ldapi.AuditLogEntryListingRep{}, transformers.WithSkipFields("Id", "Date"), transformers.WithSkipFields("Links")),
+		Columns: schema.ColumnList{
+			{
+				Name:     "id",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("Id"),
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
+				},
+			},
```

Comment:
Is there a reason why you can't use `transformers.WithPK()`?

Reply:
Yes, the json tag is `_id` which leads to a column with an underscore. Which is common with some LD fields, but it wasn't great having the ID field like that. Maybe could remove all preceding `_` chars from fields.

Reply:
Fair! Definitely not worth investigating more field name processing